atlas_cmake/package_filters.txt

```diff
@@ -1,3 +1,6 @@
+# simple_project normal C++ but setup to work with an ATLAS Analysis Release layout
 + simple_project
+# nlohmann_example is an example project that involves an external Git submodule
++ nlohmann_example
 #
-# - .*
+- .*
```

Comment:
The issues in https://github.com/matthewfeickert/atlas-cmake-nanobind-example/pull/10#issuecomment-2576572464 with not finding a `Makefile` were due to `- .*` not being used to remove all other possible package targets(?).